docs/USAGE-GUIDE.md

```diff
@@ -791,9 +791,25 @@ Authentication is usually the most expensive part of the handshake. To avoid the
 
 ### Configuring the Trust Store
 
-To validate the peer’s certificate, the local “trust store” must contain a certificate that can authenticate the peer’s certificate.
+To validate the peer’s certificate, a config's local “trust store” must contain a certificate
+that can authenticate the peer’s certificate.
 
-By default, s2n-tls will be initialized with the common trust store locations for the host operating system. To completely override those locations, call `s2n_config_wipe_trust_store()`. To add certificates to the trust store, call `s2n_config_set_verification_ca_location()` or `s2n_config_add_pem_to_trust_store()`.
+Configs created with `s2n_config_new_minimal()` contain empty trust stores, and are not able to
+validate any certificates by default. To add certificates to the trust store, call
+`s2n_config_set_verification_ca_location()` or `s2n_config_add_pem_to_trust_store()`.
+
+Operating systems typically install a set of default system certificates used by TLS clients to
+verify the authenticity of public TLS servers. If s2n-tls is operating as a client connecting to
+such a server, `s2n_config_load_system_certs()` can be used to load the system default certificates
+into the trust store. Alternatively, a config can be created with `s2n_config_new()`, which loads
+the system certificates into the trust store by default.
+
+If s2n-tls is operating as a server, or is a client connecting to a non-public TLS server, the
+certificates issued to public TLS servers should not be trusted. In these scenarios, 
+`s2n_config_new_minimal()` should be used when creating configs to avoid loading system 
```

Comment:
```suggestion
`s2n_config_new_minimal()` should be used when creating configs as there is no need to trust public TLS certificates.
```
You've already explained what s2n_config_new_minimal does so I don't think it's necessary to repeat yourself here.

Reply:
Also do you think we should mention that s2n_config_new takes longer to execute than minimal?